source/tyr/Procfile

```diff
@@ -1,3 +1,4 @@
 web: ./manage_tyr.py runserver
 worker: celery worker -A tyr.tasks
 scheduler: celery beat -A tyr.tasks
+reloader_at: ./manage_tyr.py at_reloader
```

Comment:
I just followed the doc : https://github.com/CanalTP/navitia/blob/dev/source/tyr/readme.md#example-with-honcho
If it's no use, I can delete it ^^